config/manager/manager.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         - /manager
         args:
         - --leader-elect=false
+        - --job-cleanup-on-success-delay=10
```

Comment:
I think this is causing your CI failures on the e2e tests

Reply:
I think I found the source of this issue -- see the comments on deletion, we are queueing another reconcile when we update the DeleteAfter.